CartesianPoint.java

```diff
@@ -48,25 +50,31 @@ public double getDistance(double oX, double oY) {
         double distX = oX - this.x;
         double distY = oY - this.y;
         return Math.sqrt((distX * distX) + (distY * distY));
+    }
 
+    public double getDistance(CartesianPoint point2){
+        double distX = point2.getX() - this.x;
```

Comment:
a common parameter name for this kind of method is `other`, doesn't matter though